versions.tf

```diff
@@ -3,7 +3,7 @@
 
 terraform {
 
-  required_version = ">= 1.0.0"
+  required_version = ">= 1.4.0"
```

Comment:
Be aware this might be a breaking change for some folks as they run on systems where tf 1.0.0 has been installed

Reply:
Expected this to be ">= 1.4.5" since that is the version you tested. Potential risk, even if low to leave it as 1.4.0 since there are 1.4.x versions prior to 1.4.5

Reply:
Updated in latest change.